environment/src/main/resources/application.yml

```diff
@@ -59,6 +59,8 @@ environment:
   freeipa:
     url: http://localhost:8090
     support: AWS,AZURE
+  sdx:
+    url: http://localhost:8086/dl
```

Comment:
the context path like /dl should be separated from the url

Reply:
ah you fixed in case of freeipa yesterday, hm